packages/herdtools7/herdtools7.7.54/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+synopsis: "The herdtools suite for simulating and studying weak memory models"
+maintainer: "Luc Maranget <Luc.Maranget@inria.fr>"
+authors: [
+  "Luc Maranget <Luc.Maranget@inria.fr>"
+  "Jade Alglave <j.alglave@ucl.ac.uk>"
+  "Vincent Jacques <vincent@russian-dolls-sunflowers.com>"
+]
+homepage: "http://diy.inria.fr/"
+bug-reports: "http://github.com/herd/herdtools7/issues/"
+doc: "http://diy.inria.fr/doc/index.html"
+dev-repo: "git+https://github.com/herd/herdtools7.git"
+build: ["./dune-build.sh" "%{prefix}%"]
```

Comment:
I looked at the code and I'm not too sure what all this shell boilerplate is about

Reply:
Also, I've tried your package locally and it doesn't work. It just installs empty directories

Reply:
I've tried to make it work using dune + a simple configure script to send a PR back to you but I'm not sure to understand what you wanted installed in the first place. What executables do you want to install? Is `herd` a library or an executable?

Reply:
I am working on it. I do not understand what happens either.
To answer your question, the intend is to install several exectables (herd7,litmus7 etc.) in PREFIX/bin, as well as files in PREFIX/share.

Reply:
I'm pretty sure you can do all that with dune without any shell scripts

Reply:
It may be the case that one can do all that with dune, but I do not know how to proceed...